services/src/main/java/io/grpc/services/CallMetricRecorder.java

```diff
@@ -65,16 +66,16 @@ public static CallMetricRecorder getCurrent() {
   }
 
   /**
-   * Records a call metric measurement for utilization.
-   * If RPC has already finished, this method is no-op.
+   * Records a call metric measurement for utilization in the range [0, 1]. Values outside the valid
+   * range are rejected. If RPC has already finished, this method is no-op.
```

Comment:
"rejected" isn't precise for the behavior your describing. I'd expect rejected valued to throw an exception. I think you want "ignored."

Reply:
Done. Updated JavaDocs comments to "ignored` for all relevant methods.